pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -28,7 +28,12 @@ social:
 -# We set the DCDO default to "soon-hoc" here because we can't change the DCDO flag on the test machine, but ui tests rely on hourofcode.com being in a hoc_mode consistent with production. NOTE: Erin B. needs to update this default when we flip to "actual-hoc".
 -hoc_mode = DCDO.get("hoc_mode", "soon-hoc")
 
--HIGHLIGHTS_CONSTANTS = [{:id=>45, :heading_s=>hoc_s(:highlights_constant_alba_heading), :short_heading_s=>hoc_s(:highlights_constant_alba_short_heading), :image_url_s=>"jessica.jpg", :link_text_s=>hoc_s(:highlights_facebook), :url_s=>"https://www.facebook.com/Code.org/photos/pb.309754825787494.-2207520000.1453330310./795469473882691/"}, {:id=>16, :heading_s=>hoc_s(:highlights_constant_obama_heading), :short_heading_s=>hoc_s(:highlights_constant_obama_short_heading), :image_url_s=>"obama-fistbump.jpg", :link_text_s=>hoc_s(:highlights_watchvideo), :url_s=>"https://youtu.be/AI_dayIQWV4"}, {:id=>13, :heading_s=>hoc_s(:highlights_constant_malala_heading), :short_heading_s=>hoc_s(:highlights_constant_malala_short_heading), :image_url_s=>"video-malala.png", :link_text_s=>hoc_s(:highlights_watchvideo), :url_s=>"https://youtu.be/cFdncBMDtP8"}, {:id=>64, :heading_s=>hoc_s(:highlights_constant_trudeau_heading), :short_heading_s=>hoc_s(:highlights_constant_trudeau_short_heading), :image_url_s=>"hoc-trudeau.jpg", :link_text_s=>hoc_s(:highlights_watchvideo), :url_s=>"https://www.youtube.com/watch?v=8QwT58VTj_g"}, {:id=>63, :heading_s=>hoc_s(:highlights_constant_ny_heading), :short_heading_s=>hoc_s(:highlights_constant_ny_short_heading), :image_url_s=>"chicago-2.png", :link_text_s=>hoc_s(:highlights_readmore), :url_s=>"http://www.brooklynpaper.com/stories/39/5/dtg-hour-of-code-challenge-2016-01-29-bk.html"}, {:id=>65, :heading_s=>hoc_s(:highlights_constant_athletes_heading), :short_heading_s=>hoc_s(:highlights_constant_athletes_short_heading), :image_url_s=>"athletes.png", :link_text_s=>hoc_s(:highlights_watchvideo), :url_s=>"https://code.org/athletes"}]
+:ruby
+  HIGHLIGHTS_CONSTANTS = [
+    {:id=>64, :heading_s=>hoc_s(:highlights_constant_trudeau_heading), :short_heading_s=>hoc_s(:highlights_constant_trudeau_short_heading), :image_url_s=>"hoc-trudeau.jpg", :link_text_s=>hoc_s(:highlights_watchvideo), :url_s=>"https://www.youtube.com/watch?v=8QwT58VTj_g"},
+    {:id=>63, :heading_s=>hoc_s(:highlights_constant_ny_heading), :short_heading_s=>hoc_s(:highlights_constant_ny_short_heading), :image_url_s=>"chicago-2.png", :link_text_s=>hoc_s(:highlights_readmore), :url_s=>"http://www.brooklynpaper.com/stories/39/5/dtg-hour-of-code-challenge-2016-01-29-bk.html"},
+    {:id=>65, :heading_s=>hoc_s(:highlights_constant_athletes_heading), :short_heading_s=>hoc_s(:highlights_constant_athletes_short_heading), :image_url_s=>"athletes.png", :link_text_s=>hoc_s(:highlights_watchvideo), :url_s=>"https://code.org/athletes"}
```

Comment:
Nit: prefer new format over hash rocket.  These lines are pretty long, we could format them into one key per line.